website/docs: Add docs for webauthn hints feature (#20933)
goauthentik/authentik: website/docs/add-secure-apps/flows-stages/stages/authenticator_webauthn/index.mdx

Configure if authentik will require either a removable device (like a YubiKey, Google Titan, etc) or a non-removable device (like Windows Hello, TouchID or password managers), or not send a requirement.

#### WebAuthn Hints

> :::info

Reviewer: let's add a title to this info box

> Optional hints to guide the browser in prioritizing the preferred authenticator type during registration. The available hints are:

Reviewer 2: Because this content is exactly the same (?) except for the last paragraph, we could use an <import> and have this content in one place, then pull it in to these two places. Up to you Marcelo, but if this content is likely to change, I'd recommend (haha I'd suggest) that we use an import.

- **Security key**: Suggests the user authenticate with a portable FIDO2 device such as a YubiKey.
- **Client device**: Suggests the user authenticate with a built-in platform authenticator such as Touch ID or Windows Hello.
- **Hybrid**: Suggests the user authenticate using a platform authenticator on a nearby mobile device, typically via a QR code.

> The order of selected hints matters: the first hint has the highest priority. For example, selecting "Security key" first and "Hybrid" second tells the browser to prefer security keys but offer hybrid as a fallback.

Reviewer: ```suggestion
The order of selected hints matters: the first hint has the highest priority. For example, selecting "Security key" first and "Hybrid" second asks the browser to prefer security keys before hybrid registration.
```

- **Client device**: Suggests that the user register a credential with a built-in platform authenticator such as Touch ID or Windows Hello.
- **Hybrid**: Suggests that the user register a credential using a platform authenticator on a nearby mobile device, typically via a QR code.

The order of selected hints matters: the first hint has the highest priority. For example, selecting "Security key" first and "Hybrid" second asks the browser to prefer security keys before hybrid registration.

> For backward compatibility with older browsers that do not support hints, authentik automatically infers the `authenticatorAttachment` parameter from the selected hints when it is not explicitly set:

Reviewer: can we point to some docs on what that param is
